sdk/scheduler/src/main/java/com/mesosphere/sdk/curator/CuratorStateStore.java

```diff
@@ -448,6 +452,38 @@ private String getTasksRootPath() {
         }
     }
 
+    @SuppressFBWarnings("UC_USELESS_OBJECT")
```

Comment:
Should add a comment explaining the scenario and what needs to be done to fix it (copy/paste from PR desc)